string NAME=descend

ObjectDrone d
ObjectLocation from
ObjectLocation to
